src/schnetpack/nn/embedding.py

```diff
@@ -280,25 +293,24 @@ def reset_parameters(self) -> None:
 
     def forward(
         self,
-        atomic_features: torch.Tensor,
-        electronic_feature: torch.Tensor,
-        num_batch: int,
-        batch_seg: torch.Tensor,
-        eps: float = 1e-8,
+        embedding,
```

Comment:
this could be renamed to prev embedding, to make it more clean?

Reply:
maybe `input_embedding`?

Reply:
I also like `input_embedding`